docs/user-guide/gpus.md

```diff
@@ -0,0 +1,136 @@
+---
+assignees:
+- vishh
+title: GPU Support
+---
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
+This page describes how users can consume GPUs and the current limitations.
+
+## Pre-requisites
+
+1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise.
+2. A special **alpha** feature gate `Accelerators` has to be set to true across the system: `--feature-gates="Accelerators=true"`.
```

Comment:
I looked for a doc on feature gates and there isn't one :disappointed: 
Each k8s provider chooses to expose flags in their own manner. For GKE, an "alpha" cluster would turn on this feature automatically. I plan on documenting this in the GKE side. 
I don't know what to do with other providers.
We need a doc on "feature gates" in general.